src/components/reports/ReportForm.test.tsx

```diff
@@ -16,7 +17,7 @@ type DropDown = {
 };
 
 const setDummyValues = (withDates: boolean, dropDowns?: DropDown[]) => {
-  const dummyValue = "Always the same";
+  const dummyValue = "Always_the_same";
```

Comment:
Oh I might have led you stray, try this and see if the CI passes
```suggestion
  const dummyValue = "always_the_same";
```